File: mobile-sdk/ios/ui-customization/objectivec/ui-customization.m
// Obtain an instance of the desired theme
DojoThemeSettings *lightTheme = [DojoThemeSettings getLightTheme];
DojoThemeSettings *darkTheme = [DojoThemeSettings getDarkTheme];

// Set an individual property to a custom color
lightTheme.primaryLabelTextColor = [UIColor red];

// Pass the desired theme to the SDK when starting the payment flow
DojoSDKDropInUI *dojoUI = [[DojoSDKDropInUI alloc] init];
[dojoUI startPaymentFlowWithPaymentIntentId:@"-" themeSettings:darkTheme];

File: mobile-sdk/ios/ui-customization/swift/ui-customization.swift
  // Obtain an instance of the desired theme
  var lightTheme = DojoThemeSettings.getLightTheme()
  let darkTheme = DojoThemeSettings.getDarkTheme()

  // Set an individual property to a custom color
  lightTheme.primaryLabelTextColor = .red

  // Pass the desired theme to the SDK when starting the payment flow
  let dojoUI = DojoSDKDropInUI()
  dojoUI.startPaymentFlow(themeSettings: darkTheme)